models/patientsymptom.model.js

```diff
@@ -0,0 +1,73 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const patientsymptomSchema = new Schema({
+
+    patientFullName: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    date: {
+        type:Date,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom1: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom2: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom3: {
+        type: String,
+        //required: true,
+        
+       // unique: true
+    },
+    symptom4: {
+        type: String,
+      //  required: true,
+        
+       // unique: true
+    },
+    additionalNote: {
+        type: String,
+       // required: true,
+        
+        //unique: true
+    },
+    temperature: {
+        type: Number,
+        //required: true,
+        
+        //unique: true
+    },
+     /*
+      media: { 
+          type: Buffer,
+        
+        
+        unique: true
+    },
+   
+  
+    doctorNote: {
+        type: String
+    },
+    */
+   immediateAttention: {
+        type: Boolean
+    } 
+});
```

Comment:
Per today's conversation, we need to have a PatientId in our Patient Entry/Symptom schema, so this schema needs to be updated

Reply:
so I need to create a relation with patient Schema, so I will get the patient ID from there